monai/data/iterable_dataset.py

```diff
@@ -116,7 +116,7 @@ def randomize(self, size: int) -> None:
         self._idx = self.R.randint(size)
 
     def set_random_state(self, seed: Optional[int] = None, state: Optional[np.random.RandomState] = None):
-        raise NotImplementedError(f"`set_random_state` is not available in {self.__class__.__name__}.")
```

Comment:
Should just remove this method altogether since it doesn't do anything to the inherited one.

Reply:
that's right, let me update this..